src/components/DBSearchResults.vue

```diff
@@ -162,7 +162,10 @@
           <hr>
 
           <h3 class="mt-3 mb-2 fs-5">Writer(s)</h3>
-          <p class="m-3">{{getCrewMember(topStructure(result).crew, "Writer")}}</p>
+          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Writer").length'>{{getCrewMember(topStructure(result).crew, "Writer")}} (Writer)</p>
+          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Screenplay").length'>{{getCrewMember(topStructure(result).crew, "Screenplay")}} (Screenplay)</p>
+          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Story").length'>{{getCrewMember(topStructure(result).crew, "Story")}} (Story)</p>
+          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Novel").length'>{{getCrewMember(topStructure(result).crew, "Novel")}} (Novel)</p>
```

Comment:
```suggestion
          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Writer")'>{{getCrewMember(topStructure(result).crew, "Writer")}} (Writer)</p>
          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Screenplay")'>{{getCrewMember(topStructure(result).crew, "Screenplay")}} (Screenplay)</p>
          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Story")'>{{getCrewMember(topStructure(result).crew, "Story")}} (Story)</p>
          <p class="m-3" v-if='getCrewMember(topStructure(result).crew, "Novel")'>{{getCrewMember(topStructure(result).crew, "Novel")}} (Novel)</p>
```
If the return value of `getCrewMember` were an Array you would need to check if it had length in order to know if the array has any values in it. Since the return value is always a String, you can just check if they exist because the worst case scenario is `""` which is falsey in javascript.

https://developer.mozilla.org/en-US/docs/Glossary/Falsy

As a convenience, I've made this as a suggestion on Github so, if you approve of my suggestion, you can just apply it right here and then ask for review again (right hand column) without needing to do any coding on your machine.

Reply:
For some reason, it will not allow me to apply the suggestion. Maybe because I did something else on my same branch but haven't committed it yet? Anyway, I made the changes on my machine, along with some more. I will submit a new pull request.